### DISCUSSION TITLE
Doubt About Multiple Prompts - Fabio Pantoja

### DISCUSSION THREAD

**Discussion by ProfessorPantoja** [lllyasviel/Fooocus#1572] [General]
I have a question, and this is my first time participating here on the forum.

I used the following prompt: ''Masterpiece, A __Animais__" In 'animals,' I included 5 animals: cat, dog, bird, lion, snake. However, when generating, it repeats the same animal three times. I imagine this is the standard behavior, randomizing each line within the text file. Yet, I believe having a checkbox to allow us to choose whether we want to randomize or not would be the best approach. In my specific case, I would like it to generate one of each animal.

I have a question: How can I make it generate one of each animal?

Thank you for your attention, and stay peaceful.

![image](https://github.com/lllyasviel/Fooocus/assets/123105983/01f2086d-7b71-4a60-b30e-3d1d80ef79cb)

**Comment by xhoxye** [comment]
I have the same idea and wish there was an option that would decide if it was random or sequential

**Comment by xhoxye** [comment]
I used wildcards to do that.
https://github.com/lllyasviel/Fooocus/pull/1761

**Comment by ProfessorPantoja** [comment]
> I used wildcards to do that.
> https://github.com/lllyasviel/Fooocus/pull/1761

How?

**Reply by xhoxye** [reply]
https://github.com/xhoxye/Fooocus-up_mask/raw/Fooocus_wildcards_order_read/webui.py
https://github.com/xhoxye/Fooocus-up_mask/raw/Fooocus_wildcards_order_read/modules/async_worker.py
https://github.com/xhoxye/Fooocus-up_mask/raw/Fooocus_wildcards_order_read/modules/sdxl_styles.py
Replace the original file with the file I modified, check it on startup, and write your prompt line to a wildcard file, and then call the wildcard file in the prompt. Set the number of images generated at once to 5.
prompt:  Masterpiece, `__`Animais`__`
wildcards\Animais.txt
cat
dog
bird
lion
snake
